### DISCUSSION TITLE
FPS drops/Lags on DX11

### DISCUSSION THREAD

**Discussion by squareSphere29** [ppy/osu#23775] [Q&A] [answered]
### Type

Performance

### Bug description

There is lag when opening the Mod Selection window.

Cycling through song selection causes fps drops.

Entering the end score screen after a song also lags.

Switching to OpenGL renderer improves performance. Changing GPU (integrated/dedicated) on AMD/Nvidia control panel does not improve it.

Issue does not exist prior to May2023 update (before removal of Vulkan?) - I cant recall properly when it started

### Screenshots or videos

DX11
https://github.com/ppy/osu/assets/18124696/44d87715-c0be-4176-955c-50b0dcc0ecbe

OpenGL
https://github.com/ppy/osu/assets/18124696/1b8ab388-1ba1-4170-b5cf-48010601d6e0

### Version

2023.0605.0-lazer

### Logs

DX11
[database.log](https://github.com/ppy/osu/files/11663974/database.log)
[input.log](https://github.com/ppy/osu/files/11663975/input.log)
[network.log](https://github.com/ppy/osu/files/11663977/network.log)
[performance.log](https://github.com/ppy/osu/files/11663978/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11663980/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11663981/updater.log)

OpenGL
[database.log](https://github.com/ppy/osu/files/11663959/database.log)
[input.log](https://github.com/ppy/osu/files/11663960/input.log)
[network.log](https://github.com/ppy/osu/files/11663961/network.log)
[performance.log](https://github.com/ppy/osu/files/11663962/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11663963/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11663964/updater.log)

**Comment by peppy** [comment, marked as answer]
Thanks for your feedback and videos. 

We are aware of some performance issues with direct x and will continue to optimise it.

Until then, we would recommend using legacy Open GL if it works better for you.
